### PR TITLE
Update pre-req section with image and network pre-req

### PR DESCRIPTION
**Pull request by dpkshetty**
Update pre-req section with rhcos, rhel image and subnet name pattern expected by the install script

### CODE REVIEW

**Review by yussufsh — COMMENTED** [review]
You may want to remove this text from top "The script assumes PowerVS prerequisites for OpenShift are already in place." Or move these links on top.

**Comment by dpkshetty** [comment]
> You may want to remove this text from top "The script assumes PowerVS prerequisites for OpenShift are already in place." Or move these links on top.

@yussufsh good point!, Done.

File: README.md [hunk 1]
```diff
@@ -117,13 +117,16 @@ Submit issues at: https://github.com/ocp-power-automation/openshift-install-powe
 ```
 
 ## Prerequisites
-The script assumes PowerVS prerequisites for OpenShift are already in place.
+**The script assumes PowerVS prerequisites for OpenShift are already in place.**<br>
+In case you missed, here are the links to the prerequisites:
+1. For setting up the network in Power Virtual Server service, please refer this [link](https://developer.ibm.com/tutorials/setting-up-powervs-for-ocp/)
+2. For instructions on how to prepare RHEL & RHCOS OVA images, please refer this [link](https://developer.ibm.com/tutorials/preparing-ova-for-openshift-on-cloud/)
```
Comment (bpradipt) [inline README.md:123]: How about just linking to `https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/docs/ocp_prereqs_powervs.md` ?
The blogs can go stale very soon and I think its better to the link the github readme ..

Comment (dpkshetty) [reply README.md]: @bpradipt fair point.. update the patch to reflect the same.

Comment (bpradipt) [reply README.md]: thanks @dpkshetty. 
The patch lgtm. However before merge, can you please squash the commits into a single one, add SOB and re-push.

**Comment by dpkshetty** [comment]
@bpradipt I use Github web UI and I couldn't find a easy way to squash the commits :) 
Easier was to create another PR #107 
I will close this one

**Pull request closed without merging**